src/ocm/components/clusters/utils.ts

```diff
@@ -50,3 +50,10 @@ export const canAbortInstallation = (cluster: Cluster) => {
   ];
   return allowedClusterStates.includes(cluster.status);
 };
+
+export const isSomeDisksSkipFormatting = (cluster: Cluster) => {
```

Comment:
Either use `some` or `find` since you don't care about how many there are, or which they are.
`filter` is less efficient as it iterates over all of the elements.

That probably fixes as well the issue that the function either returns  `undefined` or `boolean` and there is an Estlint warning about it.

Reply:
Sorry, the issue about the return type is still present.
This function should always return a `boolean` and now it either returns `undefined` or a `Host`. 

Probably `some` is a better option as you don't need to convert anything.
![return type](https://user-images.githubusercontent.com/829045/187924369-98f66fde-4f93-4a4f-a8f3-514e92672be4.png)

Reply:
Changed it

Reply:
I think there is a regression here, the warning appears again.
The fixed version used `some` I remember (it should always return a boolean)

Reply:
True. Problem with rebase. Solved